gen.go

```diff
@@ -308,18 +308,15 @@ func fetchISO() (map[string]*currencyInfo, error) {
 
 	currencies := make(map[string]*currencyInfo, 170)
 	for _, entry := range aux.Table[0].Entry {
-		if entry.Code == "" || entry.Name.IsFund {
+		if entry.Code == "" || entry.Number == "" || entry.Digits == "N.A." {
 			continue
 		}
-		if entry.Code == "XUA" || entry.Code == "XSU" || entry.Code == "XDR" {
-			continue
-		}
-		if strings.HasPrefix(entry.Country, "ZZ") {
-			// Special currency (Gold, Platinum, etc).
-			continue
+
+		digits, err := strconv.Atoi(entry.Digits)
```

Comment:
This change doesn't actually improve the logic because the digits are never used, they are replaced via replaceDigits() in the next call. So it's fine to keep using parseDigits(), possibly with a new comment explaining the situation.